ivy/functional/frontends/paddle/tensor/creation.py

```diff
@@ -114,3 +114,15 @@ def tril(x, diagonal=0, name=None):
 @to_ivy_arrays_and_back
 def triu(x, diagonal=0, name=None):
     return ivy.triu(x, k=diagonal)
+
+
+@with_supported_dtypes({"2.5.0 and below": ("int32", "int64")}, "paddle")
+@to_ivy_arrays_and_back
+def triu_indices(row, col=None, offset=0, dtype="int64"):
+    try:
+        arr = ivy.triu_indices(row, col, offset)
+        arr = ivy.astype(arr, ivy.as_native_dtype(dtype))
+        return arr
+    except Exception as e:
+        print("Generic Error: ", e)
+        return arr

```

Comment:
We shouldn't be putting our code in a try block and catching the exceptions, instead let them propagate to the user.